examples/quickstart/main.rs

```diff
@@ -186,6 +187,11 @@ async fn main() {
                     tvl_filter.clone(),
                     Some(uniswap_v4_pool_with_hook_filter),
                 )
+                .exchange::<UniswapV4State>(
+                    "angstrom",
+                    tvl_filter.clone(),
+                    Some(angstrom_pool_filter),
+                )
```

Comment:
I wasn't thinking of modelling angstrom as a separate exchange, instead it is just part of v4. Solvers can filter/include by hooks by modifying this filter when they subscribe.